StripeIdentity/StripeIdentity/Source/NativeComponents/Views/IdentityHTMLView/HTMLTextView.swift

```diff
@@ -98,10 +137,7 @@ final class HTMLTextView: UIView {
         DispatchQueue.main.async { [weak self] in
             do {
                 // Recompute attributed text with updated font sizes
```

Comment:
this comment now seems out of date?

Reply:
It's actually still doing the samething inside `configure`, updated the comment a bit